src/main/java/gregtech/api/recipes/logic/ParallelLogic.java

```diff
@@ -33,11 +33,7 @@ public static int getMaxRecipeMultiplier(Recipe recipe, IItemHandlerModifiable i
         int itemMultiplier = getMaxRatioItem(ingredientStacks, recipe, parallelAmount);
         // Find the maximum number of recipes that can be performed from the fluids in the fluid input inventories
         int fluidMultiplier = getMaxRatioFluid(fluidStacks, recipe, parallelAmount);
-        // If both fail to return a valid amount
```

Comment:
There should still be a safety check here:
```java
if(itemMultiplier == Integer.MAX_VALUE && fluidMultiplier == Integer.MAX_VALUE) {
    return 0;
}
```
Just to catch any weirdness at all with the recipe